From 7df3d554c7d550de5a771d6e195e99ee5ea84467 Mon Sep 17 00:00:00 2001
From: Toma Tabacu <toma.tabacu@imgtec.com>
Date: Mon, 22 Jun 2015 12:36:15 +0100
Subject: [PATCH] MIPS: LLVMLinux: Add missing comma to 2 .type assembler
 directives.

The documented syntax for the gas '.type' directive requires a comma between
the name of the type and the type description.

Apparently, gas also supports not including the comma.
The llvm integrated assembler does not support this undocumented extension.

The changed code is successfully assembled by both gas and the llvm ias.

Signed-off-by: Toma Tabacu <toma.tabacu@imgtec.com>
---
 arch/mips/kernel/r2300_fpu.S | 2 +-
 arch/mips/kernel/r4k_fpu.S   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/kernel/r2300_fpu.S b/arch/mips/kernel/r2300_fpu.S
index 5ce3b74..47dafee 100644
--- a/arch/mips/kernel/r2300_fpu.S
+++ b/arch/mips/kernel/r2300_fpu.S
@@ -125,7 +125,7 @@ LEAF(_restore_fp_context)
 	END(_restore_fp_context)
 	.set	reorder
 
-	.type	fault@function
+	.type	fault,@function
 	.ent	fault
 fault:	li	v0, -EFAULT
 	jr	ra
diff --git a/arch/mips/kernel/r4k_fpu.S b/arch/mips/kernel/r4k_fpu.S
index 1d88af2..123298b 100644
--- a/arch/mips/kernel/r4k_fpu.S
+++ b/arch/mips/kernel/r4k_fpu.S
@@ -276,7 +276,7 @@ LEAF(_restore_fp_context32)
 
 	.set	reorder
 
-	.type	fault@function
+	.type	fault,@function
 	.ent	fault
 fault:	li	v0, -EFAULT				# failure
 	jr	ra
-- 
1.9.1

